src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Avoid disabling eslint rules as much as possible. If you have to, it's preferable to do it for a single line, rather than for the entire file.